pykickstart/commands/autopart.py

```diff
@@ -328,6 +430,15 @@ def parse(self, args):
         return retval
 
 class F21_AutoPart(F20_AutoPart):
+    """
+        ``autopart [--encrypted [--passphrase=<phrase>]
+        [--escrowcert=<url> [--backuppassphrase=<passwd>]]
+        --cipher=<cipher>]
+        [--nolvm] | [--type=<type>] [--fstype=<fstype>]``
+
+        where <type> is lvm, btrfs, plain, thinp,
+        <fstype> is ext4, xfs, etc.
```

Comment:
Likewise here, with some additional formatting to line up <fstype> under <type>.  Assuming that can be done.